onmt/bin/server.py

```diff
@@ -75,14 +75,17 @@ def translate():
         inputs = request.get_json(force=True)
         out = {}
         try:
-            trans, scores, n_best, times = translation_server.run(inputs)
+            trans, scores, n_best, _, aligns = translation_server.run(inputs)
```

Comment:
Not sure why we returned the times in the first place, since we're logging translation time inside the function already. @pltrdy was there a reason or can we simplify and remove this from the return?